components/public-api/gitpod/experimental/v1/workspaces.proto

```diff
@@ -86,21 +89,20 @@ message CreateAndStartWorkspaceResponse {
 }
 
 message StartWorkspaceRequest {
-    string idempotency_token = 1;
```

Comment:
@easyCZ I wonder is it fine removing it? I remember it was rather opinionated changes at the beginning. 

We don't trying to finalize APIs but just improve VS Code experience, from other side it would be good to move in right direction.

Reply:
we don't need it here, as the workspaceId is the defacto idempotency_token

Reply:
So the original intention was that the client generates an ID, which it then attaches to all requests in the given context. This was designed (largely) to prevent a Create loop to create multiple workspaces when it's in fact just a retry, or bad client behaviour.

On Start, it makes a bit less sense, as the API should either restart a workspace, or it should reject (or perhaps just succeed) when the workspace is already starting/running.

Reply:
However, for compatibility you must not remove it, but instead mark the field as reserved.

Reply:
In this case, I think it's safe to remove.